src/main/java/temporal/ChildWorkflowImpl.java

```diff
@@ -0,0 +1,28 @@
+package temporal;
+
+import io.temporal.workflow.Workflow;
+import java.time.Duration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import temporal.model.OrderInfo;
+
+// Child Workflow Implementation
+public class ChildWorkflowImpl implements ChildWorkflow {
+  private static final Logger logger = LoggerFactory.getLogger(ChildWorkflowImpl.class);
```

Comment:
Replace `LoggerFactory.getLogger` with `Workflow.getLogger` - the Workflow logger should be used within Workflow classes

Reply:
@reselbob please address this

Reply:
Activities too? @pvsone

Reply:
Addressed: Workflows and activities. Sorry if I missed it. Some conversations were collapsed. @pvsone